Mathlib/Data/Nat/ModEq.lean

```diff
@@ -532,4 +541,5 @@ theorem odd_mod_four_iff {n : ℕ} : n % 2 = 1 ↔ n % 4 = 1 ∨ n % 4 = 3 :=
     fun h => Or.elim h odd_of_mod_four_eq_one odd_of_mod_four_eq_three⟩
 #align nat.odd_mod_four_iff Nat.odd_mod_four_iff
 
-end Nat
+lemma mod_eq_of_modEq {a b n} (h : a ≡ b [MOD n]) (hb : b < n) : a % n = b := by
+  simpa [show a % n = b % n from h] using mod_eq_of_lt hb
```

Comment:
I assume this is `h.trans (mod_eq_of_lt hb)`?

Reply:
I changed to `Eq.trans h (mod_eq_of_lt hb)`